### DISCUSSION TITLE
SPNEGO Support

### DISCUSSION THREAD

**Discussion by kd303** [dexidp/dex#1892] [Ideas]
Hi All,

thank you for the work that you guys are putting in. Can you please let me know whether SPNEGO (GS API) is supported for login into SPNEGO.

we have deployed kuberenetes/kubeflow behind Dex and our security teams are bent on doing SPNEGO support to ensure SSO is provided (basically we are windows shop and use AD for all our apps for authentication) - would this be possible with Dex? If so kindly share the details/issue log etc.

**Comment by sagikazarmark** [comment]
I don't think SPNEGO is supported at the moment, but you can use an AD through LDAP.

**Reply by kd303** [reply]
What I was looking for a seemless login - may be an experience like SPNEGO or kerberos style authentication, so if I am already logged in then I should be able to go applications protected by Dex without being challenged for user/password. We have already configured with microsoft AD and now our security teams are not happy with page popping up asking for user/password, hence wondering whether this is possible in any way.
